CHANGELOG.md

```diff
@@ -11,9 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Python version `3.10` tested during CI
 - Python version `3.10` added to package classifiers
 
+### Changed
+
+- `BasicQuestion.ask()` will only evaluate dynamic values for the prompt message and default value once per instead of
```

Comment:
I think this makes the prose more clear:

```suggestion
- `BasicQuestion.ask()` will only evaluate dynamic values for the prompt message and default value once instead of
```